Doc/library/bdb.rst

```diff
@@ -352,20 +359,20 @@ Finally, the module defines the following functions:
 
 .. function:: checkfuncname(b, frame)
 
-   Check whether we should break here, depending on the way the breakpoint *b*
-   was set.
+   Return True if we should break here, depending on the way the breakpoint
+   *b* was set.
 
-   If it was set via line number, it checks if ``b.line`` is the same as the one
-   in the frame also passed as argument.  If the breakpoint was set via function
-   name, we have to check we are in the right frame (the right function) and if
-   we are in its first executable line.
+   If it was set via line number, it checks if ``b.line`` is the same as the
+   one in *frame*.  If the breakpoint was set via function name, we have to
+   check we are in the right *frame* (the right function) and if we are on
+   its first executable line.
 
 .. function:: effective(file, line, frame)
 
    Determine if there is an effective (active) breakpoint at this line of code.
-   Return a tuple of the breakpoint and a boolean that indicates if it is ok
-   to delete a temporary breakpoint.  Return ``(None, None)`` if there is no
-   matching breakpoint.
+   Return a tuple of the breakpoint that was triggered and a boolean that
+   indicates if it is ok to delete a temporary breakpoint.  Return
+   ``(None, None)`` if there is no matching breakpoint.
```

Comment:
I want to put this in the standard format.  My suggestion:
"""Return (active (*file*, *line*) breakpoint, delete temporary flag) or (None, None).

The breakpoint is the first entry in Breakpoint.bplist[file, line] (which must exist) that is enabled and has neither a false condition nor positive ignore count.  The flag, meaning that a temporary breakpoint should be deleted, is False only when the condition cannot be evaluated (in which case, ignore count is ignored).  
"""
What do you think?  The docstring for effective() should also be replaced and the obsolete pair of comment lines above the def line deleted.